docs/lago-tutorial/scenarios/jenkins/init.json

```diff
@@ -0,0 +1,82 @@
+{
+    "domains": {
+        "vm0-server": {
+            "memory": "1024",
+            "nics": [
+                {
+                    "net": "lago"
+                }
+            ],
+            "disks": [
+                {
+                    "template_name": "el7-base",
+                    "type": "template",
+                    "name": "root",
+                    "dev": "vda",
+                    "format": "qcow2"
+                }
+            ],
+            "metadata": {
+                "deploy-scripts": [
+                    "../deployment-scripts/vm0-server-deploy-script.sh"
```

Comment:
once #230 is merged, you can use $LAGO_WORKDIR_PATH here to allow relocatable workdirs and allow running the lago deploy from any dir